src/test/java/marquez/service/models/Generator.java

```diff
@@ -156,13 +160,23 @@ public static RunArgs cloneRunArgs(RunArgs ra) {
     return new RunArgs(ra.getHexDigest(), ra.getJson(), ra.getCreatedAt());
   }
 
+  // Datasource
+  public static Datasource genDatasource() {
+    int random = (int) (Math.random() * 50 + 1);
+    int random2 = (int) (Math.random() * 50 + 1);
```

Comment:
what is `random2` used for?

Reply:
Good catch. Removed in PRs #394, #387, and #384, where this is fixed.